windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -26,6 +26,11 @@ Issued:2021-07-08
 
 The WMCO 2.0.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:2671[RHBA-2021:2671].
 
+[IMPORTANT]
+====
+Users who are running WMCO 2.0.2 should first upgrade to WMCO 2.0.3 prior to upgrading to WMCO 3.0.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1983153[**BZ#1983153**])
```

Comment:
I would say any version prior to 2.0.3. This is a problem for 2.0.0 and 2.0.1 too.

Reply:
Good catch; updated.